hosts/builders/build3/configuration.nix

```diff
@@ -40,7 +40,7 @@
       {
         hostName = "hetzarm.vedenemo.dev";
         system = "aarch64-linux";
-        maxJobs = 80;
+        maxJobs = 40;
```

Comment:
You might also want to update `terraform/main.tf` to the numbers here (`ext_builder_machines`)

Reply:
Yeah, looks good!